src/components/LoginView.vue

```
@@ -0,0 +1,111 @@
+      router.push('/dashboard');
+    } catch (error) {
+      console.error('Login failed:', error);
+    }
+  };
+
+  const testConnection = async () => {
+    isPinging.value = true;
+    pingResult.value = '';
+    
+    try {
+      const response = await apiClient.get('/test/');
```

ESLint

[error] 'response' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)